src/styles/app.less

```diff
@@ -187,14 +187,14 @@ p {
     flex: 1 1 auto;
     font-size: 110%;
     color: @labelFontColor;
-    text-align: right;
+    text-align: center;
```

Comment:
Setting `text-align: center` will make the text "jump" when text length changes. I'd still prefer aligning the text on right, but maybe adding some padding or margin to the right.

Reply:
Fair point. 
See second suggestion on PR #14